article/app/views/fragments/articleAsideSlot.scala.html

```diff
@@ -0,0 +1,17 @@
+@(
+    shouldShowAds: Boolean,
+    optionalSlotSizes: Seq[String],
+    optClassNames: Option[String] = None
+
+)
+@import conf.switches.Switches.CommercialSwitch
+
+@if(CommercialSwitch.isSwitchedOn && shouldShowAds) {
```

Comment:
Somehow I feel like `CommercialSwitch.isSwitchedOn` should be moved into `Commercial.shouldShowAds`. wdyt?